lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -148,6 +148,12 @@ define(['jquery', 'underscore', 'annotator'], function ($, _, Annotator) {
                 case KEY.ESCAPE:
                     this.annotator.viewer.hide();
                     break;
+                // Gives focus to the show/hide notes button
+                case KEY.n:
+                    if (event.ctrlKey && event.altKey)  {
+                        $('.action-toggle-notes').focus();
```

Comment:
@polesye I love the idea of a dedicated shortcut for toggling notes (we should talk about different ways for informing all users that this shortcut exits.)  However I do worry a bit about choosing a key combo that doesn't collide with any native functions of either the OS, browser or screen reader.   A quick search didn't turn up anything, so I say let's give it a try!